### ISSUE TITLE
Old issue related to button hold and mouse scroll up/down.

### ISSUE THREAD

**Issue by falsonerd** [Ryochan7/DS4Windows#2788]
**Button hold keeps repeating**
I've been using DS4 Windows for remapping some keyboard buttons and mouse for quite sometime.
I read a lot of manga/web-novels online and I usually use L2 as mouse scroll up and L1 as mouse scroll down and L-Stick for mouse control. 
And since version 2.2.10 I've noticed that when you remap any button of the controller (other than L2/R2) to mouse scroll up/down and press and hold it. It won't be smooth like 2.2.9 was and you will see constant stuttering, almost like instead of holding it you were pressing it really fast.
So I've been testing each version since 2.2.9 to see if one had this issue fixed but no luck, I've been using 2.2.9 since it was the latest version that was smooth.

**To Reproduce**
1. Assign any controller button to scroll up/down
2. Press and hold the remapped button

**Expected behavior**
Smoothly scroll up ad down without stuttering.

**Desktop (please complete the following information):**
 - Controller Make and Model: Sony DS4 v.2 (CUH-ZCT2U)
 - OS:Windows 10 Pro Build 22H2
 - DS4Windows Version 2.2.10+ (same issue with all versions past it)

**Comment by Ryochan7** [comment]
That potential change will never happen. This issue got lost in the pile and there is no chance of doing anything about it now. Closing.

**Comment by Ryochan7** [comment]
I ended up looking into this anyway. Commit 6386109192f6ec2a308bc7822e6efd49ceedb38a

It looks like the behavior you were talking about was only possible with SendInput. SendInput is able to process partial mouse wheel ticks with no need to keep track of a remainder; it seems to produce a sort of smooth scrolling effect. FakerInput requires a full mouse wheel tick for any wheel movement.

I loosened up the analog to mouse wheel routine a bit more. Events will be fired more often and now there is an offset for movement. I like it although I have only played around with it in Brave browser.
